src/char_mapping.rs

```diff
@@ -94,6 +164,50 @@ table! {
     }
 }
 
+table! {
+    #[doc = "A char-to-glyph encoding in format 12."]
+    pub Encoding12 {
+        format      (u16), // format
+        reserved    (u16) = { 0 }, // reserved
+        length      (u32), // length
+        language    (u32), // language
+        num_groups  (u32), // numGroups
```

Comment:
There is one extra column of spaces before the types. Please remove it.

Reply:
Also, I usually rename fields in order to have a single style across the whole crate, and I try to avoid any kind of contractions. As far as I remember, all `num_things` fields are now `thing_count`. (That’s why I also give the original name in the comment to the right.) Let’s rename `num_groups` to `group_count`.